src/main/java/edu/harvard/iq/dataverse/harvest/server/OAIRecordServiceBean.java

```diff
@@ -303,11 +303,28 @@ public OAIRecord findOAIRecordBySetNameandGlobalId(String setName, String global
     }    
     
     public List<OAIRecord> findOaiRecordsBySetName(String setName, Date from, Date until) {
+        return findOaiRecordsBySetName(setName, from, until, false);
+    }
+    
+    public List<OAIRecord> findOaiRecordsNotInThisSet(String setName, Date from, Date until) {
+        return findOaiRecordsBySetName(setName, from, until, true);
+    }
+    
+    public List<OAIRecord> findOaiRecordsBySetName(String setName, Date from, Date until, boolean excludeSet) {
                 
         String queryString ="SELECT object(h) from OAIRecord as h where h.id is not null";
```

Comment:
The first change requested - yes, it needed to be an "OR" of course, great catch; in practical terms, the way that method was being called, the OAIrecords list was never passed to it when null. So there wouldn't be any nullpointer exceptions *right away*; but it could definitely bite us in the butt in the future. 

The second one... that "as" was definitely meaningless there. It's an old line of code too, imported from pre-Dataverse 4 implementation. So yes, I noticed and removed that "as" in line 291, but not in the other one. 
Checked in a change that makes it consistent.